sensum-pwa/src/state/sensations/State_Sensations.res

```diff
@@ -0,0 +1,21 @@
+let sensationsIndex = Recoil.asyncSelector({
```

Comment:
https://recoiljs.org/docs/basic-tutorial/selectors
and  https://recoiljs.org/docs/guides/asynchronous-data-queries#asynchronous-example